src/libs/ReportActionsUtils.js

```diff
@@ -405,6 +418,49 @@ function getLinkedTransactionID(reportID, reportActionID) {
     return reportAction.originalMessage.IOUTransactionID;
 }
 
+/**
+ * @returns {string}
+ */
+function getMostRecentReportActionLastModified() {
+    // Start with the oldest date possible
+    let mostRecentReportActionLastModified = new Date(0).toISOString();
+
+    // Flatten all the actions
+    // Loop over them all to find the one that is the most recent
```

Comment:
Is this going to be efficient and fast when you have 1 million actions?

Reply:
Probably not. But a "future us" problem maybe. I might also ask whether the app should even be allowed to hold enough data for this to become slow...

An alternative could be to have a global (to this file) `mostRecentReportActionLastModified` variable that we update whenever a report object or new report action comes in. That would be "faster", but not as obvious as this.

Other things we can do are avoid using underscore and use native array methods or for loops. But... I usually avoid that until there is a documented issue with the code.

Reply:
Would it make sense to add some "logging" (via console? something else?) to this so that it doesn't become forgotten and obscure and takes us days to figure out that it is slow?

Reply:
Definitely open to ideas. How should we implement this? Just log something to stats? Or do we want to alert if it goes over some threshold?

Reply:
Yeah send a warning log to the API if it's over like 500ms or so?

Reply:
Ok, I added something for this! It took ~5ms on my messed up local account, but you are right that it could one day prove to be slow.